src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -173,8 +196,18 @@ Status CreateRequestQueue::ProcessRequests() {
         oom_start_time_ns_ = now;
       }
 
-      // Notify the scheduler that object creation is blocked at this priority.
-      on_object_creation_blocked_callback_(queue_it->first.first);
+      if(RayConfig::instance().enable_BlockTasks()){
+        RAY_LOG(DEBUG) << "[JAE_DEBUG] calling object_creation_blocked_callback priority "
+	    << queue_it->first.first.score;
+	    on_object_creation_blocked_callback_(queue_it->first.first , true, false);
```

Comment:
We should call the callback a single time with the right flags, instead of calling it twice (this is to avoid having concurrent callbacks).